tests/TestCase/Http/Middleware/SessionCsrfProtectionMiddlewareTest.php

```diff
@@ -413,4 +413,25 @@ public function testSaltToken(): void
         }
         $this->assertCount(10, array_unique($results));
     }
+
+    /**
+     * Ensure that tokens can be regenerated
+     */
+    public function testRegenerateToken(): void
+    {
+        $request = new ServerRequest([
+            'url' => '/articles/',
+        ]);
+        $updated = SessionCsrfProtectionMiddleware::replaceToken($request);
+        $session = $updated->getSession()->read('csrfToken');
```

Comment:
Should we assert the request and updated are different and the csrfToken is different after 2 calls?